apps/storybook/src/Table.stories.tsx

```diff
@@ -570,6 +571,96 @@ Filters.args = {
   ],
 };
 
+export const GlobalFilter: Story<Partial<TableProps>> = (args) => {
+  type TableStoryDataType = {
+    name: string;
+    description: string;
+  };
+
+  const onClickHandler = (
+    props: CellProps<{ name: string; description: string }>,
+  ) => action(props.row.original.name)();
+
+  const columns = useMemo(
+    () => [
+      {
+        Header: 'Table',
+        columns: [
+          {
+            id: 'name',
+            Header: 'Name',
+            accessor: 'name',
+            Filter: tableFilters.TextFilter(),
+          },
+          {
+            id: 'description',
+            Header: 'Description',
+            accessor: 'description',
+            maxWidth: 200,
+            Filter: tableFilters.TextFilter(),
+          },
+          {
+            id: 'click-me',
+            Header: 'Click',
+            width: 100,
+            Cell: (props: CellProps<{ name: string; description: string }>) => {
+              const onClick = () => onClickHandler(props);
+              return <Anchor onClick={onClick}>Click me!</Anchor>;
+            },
+          },
+        ],
+      },
+    ],
+    [],
+  );
+
+  const data = useMemo<TableStoryDataType[]>(
+    () => [
+      { name: 'Name1', description: 'Description1' },
```

Comment:
Could you change `Name` vs `Description` numbers, eg. Name2, Description5, to add more variety. This way people can see that it is filtering both columns.